sourcemaps.js

```diff
@@ -0,0 +1,67 @@
+/* Results on Node 20.3.1, Github Actions:
+
+TODO
+*/
+
+let { existsSync, readFileSync } = require('fs')
+let { join } = require('path')
+const postcss = require('postcss')
+
+let example = join(__dirname, 'cache', 'bootstrap.css')
+let css = readFileSync(example).toString()
+
+module.exports = {
+  maxTime: 15,
+  name: 'Sourcemaps',
+  tests: [
+    {
+      defer: true,
+      fn: done => {
+        {
+          let root = postcss.parse(css, { from: example })
+          root.toResult({ map: { inline: false }, to: 'dist/bootstrap.css' }).map.toJSON()
+        }
+
+        {
+          let root = postcss.parse(css, { from: example })
+          root.toResult({ map: { absolute: true, inline: false }, to: 'dist/bootstrap.css' }).map.toJSON()
+        }
+
+        {
+          let root = postcss.parse(css, { from: example })
+          root.toResult({ map: { inline: true }, to: 'dist/bootstrap.css' })
+        }
```

Comment:
Currently running three times with various configs to make sure that most code paths are covered.